src/styles/components/_calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  width: ($day-size * $week-long) + ($cell-gap * ($week-long - 1));
+  padding: 10px;
+  gap: $cell-gap;
+  flex-wrap: wrap;
+  list-style: none;
+
+  @each $day, $day-num in $week-days {
+    &--day-start--#{$day}>:first-child {
+      margin-left: ($day-size + $cell-gap) * $day-num;
+    }
+  }
+
+  @for $day from $min-month-long through $num-days {
```

Comment:
`$min-month-long` is not necessary, because it's obvious that month has min 28 days in a year)

Reply:
:) Added this variable due to my colleague's suggestion not to hardcode numbers, didn't have it initially